src/components/utils/YodaMeta.js

```diff
@@ -0,0 +1,38 @@
+class YodaMeta {
+
+  // Sign In type
+  // 1 : Facebook
+  // 2 : LinkedIn
+  static LOGIN_TYPE_FB = '1';
+  static LOGIN_TYPE_LI = '2';
+
+   // Meta data for LinkedIn Sign In
+  static redirectUrl = 'http://localhost';
+  static clientId = '78831o5pn1vg4p';
+  static clientSecret = 'ioMEK1Qu77xqxKyu';
+  static state = 'DCEeFWf45A53qdfKef424';
+  static scopes = ['r_basicprofile', 'r_emailaddress', 'rw_company_admin'];
+
+  //Host and API
+  static host = 'http://ec2-52-78-121-221.ap-northeast-2.compute.amazonaws.com:8080/';
+  static API_LOGIN = 'users/signIn/';
+  static API_MENTOR = 'users/mentorlist/';
+  static API_ALL = 'users/all/';
+  static API_ME = 'users/me/';
+  static API_USER = 'users/id/';
+  static API_ACTIVITY = 'match/list/';
+  static API_MENTOR_REQ = 'match/request/';
+  static API_MENTOR_RESP = 'match/response/';
+
+  // Error code
```

Comment:
Hmm. It's worrisome to define a static list of these statuses. Are all these errors mutually exclusive? Couldn't we have both ERR_FB_LOGIN and ERR_LI_LOGIN? The declaration here looks like they're going to be used in a mutex-way but they can be independently set. Is this intended?